frontend/src/Components/Admin/Floaters/EditFloater.js

```diff
@@ -48,15 +48,17 @@ export default class EditFloater extends React.Component {
                     status="success"
                     message="Floater Has Been Edited"
                 />
-                <h1 className="text-center mb-3">Editing this Event</h1>
+                <h1 className="text-center mb-3 text-dark">
+                    Editing this Floater's Details
```

Comment:
Why these codes are highlighted in red? May be it is because of 's you used.

Reply:
yes @Behnazz it's because if the `'`, we can safely ignore the highlighting.